mne/viz/epochs.py

```diff
@@ -20,34 +21,46 @@
 from ..time_frequency import psd_multitaper
 from .utils import (tight_layout, figure_nobar, _toggle_proj, _toggle_options,
                     _layout_figure, _setup_vmin_vmax, _channels_changed,
-                    _plot_raw_onscroll, _onclick_help, plt_show, _handle_decim,
-                    _compute_scalings, DraggableColorbar, _setup_cmap)
+                    _plot_raw_onscroll, _onclick_help, plt_show,
+                    _compute_scalings, DraggableColorbar, _setup_cmap,
+                    _grad_pair_pick_and_name, _handle_decim)
 from .misc import _handle_event_colors
 from ..defaults import _handle_default
+from ..externals.six import string_types
 
 
 def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
                       vmax=None, colorbar=True, order=None, show=True,
-                      units=None, scalings=None, cmap='RdBu_r',
-                      fig=None, axes=None, overlay_times=None):
+                      units=None, scalings=None, cmap=None, fig=None,
+                      axes=None, overlay_times=None, combine=None,
+                      group_by=None, evoked=True, ts_args=dict()):
     """Plot Event Related Potential / Fields image.
 
     Parameters
     ----------
     epochs : instance of Epochs
         The epochs.
     picks : int | array-like of int | None
-        The indices of the channels to consider. If None, the first
-        five good channels are plotted.
+        The indices of the channels to consider. If None and `combine` is
+        also None, the first five good channels are plotted.
     sigma : float
         The standard deviation of the Gaussian smoothing to apply along
         the epoch axis to apply in the image. If 0., no smoothing is applied.
-    vmin : float
-        The min value in the image. The unit is uV for EEG channels,
-        fT for magnetometers and fT/cm for gradiometers.
-    vmax : float
-        The max value in the image. The unit is uV for EEG channels,
-        fT for magnetometers and fT/cm for gradiometers.
+        Defaults to 0.
+    vmin : None | float | callable
+        The min value in the image (and the ER[P/F]). The unit is uV for
+        EEG channels, fT for magnetometers and fT/cm for gradiometers.
+        If vmin is None and multiple plots are returned, the limit is
+        equalized within channel types.
+        Hint: to specify the lower limit of the data, use
```

Comment:
double-colon (indented parts will be code). Although here I'd probably just use inline code (double-backticks and no newline) because it's so short.